independent-publisher-connectors/Safaricom Mpesa Daraja/apiProperties.json

```diff
@@ -0,0 +1,48 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "4b048dda-9dc5-412b-b881-1f6881f882d8",
+          "scopes": [
+            "User.Read"
+          ],
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "properties": {
+            "IsFirstParty": "False",
+            "IsOnbehalfofLoginSupported": false
+          },
+          "customParameters": {
+            "authorizationUrl": {
+              "value": "https://login.microsoftonline.com/common/oauth2/authorize"
+            },
+            "tokenUrl": {
+              "value": "https://daraja.onrender.com/generate"
+            },
+            "refreshUrl": {
+              "value": "https://daraja.onrender.com/generate"
+            }
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
+    "capabilities": [],
+    "policyTemplateInstances": [
+      {
+        "templateId": "setheader",
+        "title": "add auth",
+        "parameters": {
+          "x-ms-apimTemplateParameter.name": "Authentication",
+          "x-ms-apimTemplateParameter.value": "Bearer HNwhBZTObRdSXOGUfGcUA1VnlMsD",
+          "x-ms-apimTemplateParameter.existsAction": "override",
+          "x-ms-apimTemplate-policySection": "Request"
+        }
+      }
+    ],
+    "publisher": "Bernard Karaba"
```

Comment:
@johnmuchiriM you can add both publisher here as well. Please see below.
"publisher": "Publisher1, Publisher2"

Please add the stackOwner property. Please see our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json#:~:text=%22stackOwner%22%3A%20%22Name%20of%20company%20or%20organization%20that%20connector%20is%20connecting%20to.%22). 
"stackOwner": "Name of company or organization that connector is connecting to."